### DISCUSSION TITLE
使用官方英文识别模型识别英文绘本，有些很简单的图一个都识别不出来，有些图中的I也会漏检

### DISCUSSION THREAD

**Discussion by dplcz** [PaddlePaddle/PaddleOCR#13801] [Q&A]
![page_3](https://github.com/user-attachments/assets/b0a54164-d235-409f-a5f3-5a1d8a4c3445)
这个图直接识别为空
有些识别出来又是下面的这种效果
![page_4](https://github.com/user-attachments/assets/d4df78da-c64e-49eb-8a05-e5ba9f0d1f54)

**Comment by GreatV** [comment]
paddleocr可能在这种情况下泛化性有所缺失，建议在自己的场景下微调一下OCR模型。

**Reply by dplcz** [reply]
改过一些thresh，但是没效果，请教下大佬还能调哪些参数

**Reply by GreatV** [reply]
要在自己的场景下重新训练模型
可以参考一下 https://paddlepaddle.github.io/PaddleOCR/ppocr/model_train/finetune.html
